tests/test_scheduler.py

```diff
@@ -203,6 +203,7 @@ def test_pytorch_equal_numpy(self):
 
 class DDPMSchedulerTest(SchedulerCommonTest):
     scheduler_classes = (DDPMScheduler,)
+    forward_default_kwargs = (("num_inference_steps", 50),)
```

Comment:
DDPM should not be able to change the inference steps IMO -> that's what we have DDIM for :-)